### ISSUE TITLE
[FEATURE_REQUEST] Add presets for Custom Stopping Strings

### ISSUE THREAD

**Issue by Spacellary** [SillyTavern/SillyTavern#1832]
### Have you searched for similar requests?

Yes

### Is your feature request related to a problem? If so, please describe.

Depending on the model, task the user is performing or even cards, different Stopping Sequences/Strings are useful, but not having a way to quickly set and switch between them, like Presets, is inconvenient because you need to take your previous Sequence, put it somewhere, then bring the new one, it's an unnecessary copy pasting job that can be improved in my opinion.

### Describe the solution you'd like

Presets selection/dropdown for Custom Stopping Strings with options to save them like other similar ones like in this mockup:

![image](https://github.com/SillyTavern/SillyTavern/assets/39604793/c6532c70-474b-43f9-a106-6c67cac9789f)

### Describe alternatives you've considered

Currently I use a few Lorebooks as "notes" with them, but it's just a glorified notepad at this point. Presets would be nice.

### Priority

Low (Nice-to-have)

### Is this something you would be keen to implement?

No

**Comment by Spacellary** [comment]
Yes, this is still welcome.

**Comment by Cohee1207** [comment]
I'm not sure about a dedicated preset type. We briefly discussed putting custom stopping strings under Context templates with @RossAscends. But that would certainly be a habit breaker (i.e. switching context templates would also swap stop strings).

**Comment by Spacellary** [comment]
> But that would certainly be a habit breaker (i.e. switching context templates would also swap stop strings).

Yeah... Well, that's partially why I preferred the idea of presets for those, even inside the same preset you might just want to use different stopping strings (basically catching things for for specific cards).

Having them tied to Context would work but in a roundabout way where you have multiple duplicate Context presets just for the variations in Stop Strings.

Something else is maybe making that inclusion optional or doing presets as the original idea suggests.

**Comment by Wolfsblvt** [comment]
Well, many people put in instruct-specific things in the custom stopping strings, that'd make them more related to instruct presets than context presets?
(Though I am not sure about that, considering the default stopping strings already include many instruct stopping strings you cannot change)

**Comment by Spacellary** [comment]
Either Context or Instruct would be acceptable from my point of view so I think that's also a good option.
*I was surprised it __wasn't__ already part of the presets in some way when I brought this up.*
